lochness/mediaflux/__init__.py

```diff
@@ -98,28 +99,41 @@ def sync_module(Lochness: 'lochness.config',
                                           '--mf.config', mflux_cfg,
                                           '--nb-retries 5',
                                           '--direction down', tmpdir,
-                                          mf_remote_dir,
+                                          mf_remote_root,
                                           '-o', diff_path])
-
-                        p= Popen(cmd, shell=True)
+                        
+                        p = Popen(cmd, shell=True)
                         p.wait()
 
                         # ENH
                         # if dry: exit()
-
                         if not isfile(diff_path):
                             continue
 
-                        df= pd.read_csv(diff_path)
+                        df = pd.read_csv(diff_path)
                         for remote in df['SRC_PATH'].values:
-
                             if remote is nan:
                                 continue
 
                             if not re.search(patt.replace('*','(.+?)'), remote):
                                 continue
                             else:
-                                remote= remote.split(':')[1]
+                                remote = remote.split(':')[1]
+
+                            # To keep the folder structures of the mediaflux
+                            # subpath of the data under mf_remote_root
+                            # eg) When full remote and mf_remote_root are
+                            # full remote: 
+                            # /projects/proj-5070/
+                            #   PrescientME/Interview_recordings/ME00077/
+                            #       Audio Record/OS_ZZ04139.m4a
+                            # mf_remote_root:
+                            # /projects/proj-5070/
+                            #   PrescientME/Interview_recordings/ME00077/
+                            # Then
+                            # subpath: Audio Records/OS_ZZ04139.m4a
+                            subpath = Path(remote).relative_to(
+                                    Path(mf_remote_root))
```

Comment:
Kevin, based on your comment and the use of `subpath.parent` below, I think it was already doing so before. Did you find it otherwise? 😮‍💨

Reply:
We should double check in future PRs for `Prescient` network, but I think the previous version was saving all the files under a single target directory. Again, we were assuming all the files to be saved under a single mediaflux directory in the beginning of lochness development, which we clearly stated in the SOP. However, the SOP has been updated 🙈

Reply:
Correct. We want `processed/ME00077/Interview_recordings/Audio Record/OS_ZZ04139.m4a`